### PR TITLE
Domain model update - take 2

### PR DESCRIPTION
**Pull request by atstojanov**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/cherrytwist/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/cherrytwist/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/cherrytwist/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: graphql/fragments/projectDetails.graphql [hunk 1]
```diff
@@ -3,7 +3,7 @@ fragment ProjectDetails on Project {
   textID
   name
   description
-  lifecycle {
+  lifecycle2 {
```
Comment (valentinyanakiev) [inline graphql/fragments/projectDetails.graphql:6]: lifecycle

File: graphql/queries/challengeProfile.graphql [hunk 2]
```diff
@@ -36,7 +36,7 @@ query challengeProfile($id: String!) {
           textID
           name
           description
-          lifecycle {
+          lifecycle2 {
```
Comment (valentinyanakiev) [inline graphql/queries/challengeProfile.graphql:39]: lifecycle

File: graphql/queries/projects.graphql [hunk 3]
```diff
@@ -6,7 +6,7 @@ query projects {
       textID
       name
       description
-      lifecycle {
+      lifecycle2 {
```
Comment (valentinyanakiev) [inline graphql/queries/projects.graphql:9]: lifecycle

File: src/generated/graphql.tsx [hunk 4]
```diff
@@ -74,7 +73,7 @@ export const ProjectDetailsFragmentDoc = gql`
     textID
     name
     description
-    lifecycle {
+    lifecycle2 {
```
Comment (valentinyanakiev) [inline src/generated/graphql.tsx:76]: lifecycle

File: src/generated/graphql.tsx [hunk 5]
```diff
@@ -2193,7 +2191,7 @@ export const ChallengeProfileDocument = gql`
             textID
             name
             description
-            lifecycle {
+            lifecycle2 {
```
Comment (valentinyanakiev) [inline src/generated/graphql.tsx:2194]: lifecycle

File: src/generated/graphql.tsx [hunk 6]
```diff
@@ -4158,7 +4163,7 @@ export const ProjectsDocument = gql`
         textID
         name
         description
-        lifecycle {
+        lifecycle2 {
```
Comment (valentinyanakiev) [inline src/generated/graphql.tsx:4166]: lifecycle

File: src/pages/Challenge.tsx [hunk 7]
```diff
@@ -156,7 +156,7 @@ const Challenge: FC<ChallengePageProps> = ({ paths, challenge, users = [] }): Re
         title: p?.name || '',
         description: p?.description,
         caption: p?.caption,
-        tag: { status: 'positive', text: p?.lifecycle?.state || '' },
+        tag: { status: 'positive', text: p?.lifecycle2?.state || '' },
```
Comment (valentinyanakiev) [inline src/pages/Challenge.tsx:159]: lifecycle

File: src/pages/Ecoverse.tsx [hunk 8]
```diff
@@ -111,7 +111,7 @@ const EcoversePage: FC<EcoversePageProps> = ({
           title: p?.name || '',
           description: p?.description,
           caption: parentsData?.caption,
-          tag: { status: 'positive', text: p?.lifecycle?.state || '' },
+          tag: { status: 'positive', text: p?.lifecycle2?.state || '' },
```
Comment (valentinyanakiev) [inline src/pages/Ecoverse.tsx:114]: lifecycle

File: src/pages/Project/Index.tsx [hunk 9]
```diff
@@ -33,7 +33,7 @@ const ProjectIndex: FC<ProjectPageProps> = ({ paths, project }): React.ReactElem
 
   useUpdateNavigation({ currentPaths: paths });
 
-  const { name, description, lifecycle, aspects } = project;
+  const { name, description, lifecycle2: lifecycle, aspects } = project;
```
Comment (valentinyanakiev) [inline src/pages/Project/Index.tsx:36]: lifecycle

**Review by valentinyanakiev — CHANGES_REQUESTED** [review]
lifecycle2 should stay lifecyclee

File: graphql/fragments/projectDetails.graphql (hunk 1 above)
Comment (atstojanov) [reply graphql/fragments/projectDetails.graphql:6]: The web can't control the backed.

**Review by valentinyanakiev — APPROVED** [review]

**Pull request merged**